docs/painless/painless-lang-spec/painless-types.asciidoc

```diff
@@ -25,51 +25,34 @@ force evaluation as its corresponding reference type value.
 
 The following primitive types are available:
 
-[horizontal]
-`byte`::
-8-bit, signed, two's complement integer
-* range: [`-128`, `127`]
-* default value: `0`
-* reference type: `Byte`
-
-`short`::
-16-bit, signed, two's complement integer
-* range: [`-32768`, `32767`]
-* default value: `0`
-* reference type: `Short`
-
-`char`::
-16-bit, unsigned, Unicode character
-* range: [`0`, `65535`]
-* default value: `0` or `\u0000`
-* reference type: `Character`
-
-`int`::
-32-bit, signed, two's complement integer
-* range: [`-2^31`, `2^31-1`]
-* default value: `0`
-* reference type: `Integer`
-
-`long`::
-64-bit, signed, two's complement integer
-* range: [`-2^63`, `2^63-1`]
-* default value: `0`
-* reference type: `Long`
-
-`float`::
-32-bit, signed, single-precision, IEEE 754 floating point number
-* default value: `0.0`
-* reference type: `Float`
-
-`double`::
-64-bit, signed, double-precision, IEEE 754 floating point number
-* default value: `0.0`
-* reference type: `Double`
-
-`boolean`::
-logical quantity with two possible values of `true` and `false`
-* default value: `false`
-* reference type: `Boolean`
+[[available-primitive-types]]
+.**Available primitive types**
+[%collapsible%open]
+====
+`byte` (`Byte`)::
+    8-bit, signed, two's complement integer. Range: [`-128`, `127`]. Default: `0`.
+
+`short` (`Short`)::
+    16-bit, signed, two's complement integer. Range: [`-32768`, `32767`]. Default: `0`.
+
+`char` (`Character`)::
```

Comment:
How much do the parentheticals `byte (Byte)::` add?  The only places they differ are `Character` and `Integer`.

Nothing calls out the fact that `Byte` is the reference type version of `byte`.

Reply:
Meant to expand on this, so appreciate the reminder! I'm thinking:

>The following primitive types are available. The corresponding reference type is listed in parentheses. For example, `Byte` is the reference type for the `byte` primitive type:

Reply:
works for me.